src/ensembl/src/content/app/browser/Browser.tsx

```diff
@@ -197,11 +199,9 @@ export const Browser: FunctionComponent<BrowserProps> = (
   }, [props.browserQueryParams.focus]);
 
   useEffect(() => {
-    const chrLocationForGenome = props.chrLocation[props.activeGenomeId] || [
-      '',
-      0,
-      0
-    ];
+    const chrLocationForGenome = props.chrLocation[
+      props.activeEnsObjectId[props.activeGenomeId]
```

Comment:
How did we get to the point when a prop called something-something-id is in fact a hash of ids? Should we be passing this whole hash to the Browser component? Should we not extract the relevant id in the mapStateToProps function?

(Don't know whether this grumble belongs to this PR or to a separate refactoring effort, but it just reads wrong.)

Reply:
How about renaming it to `activeEnsObjects` that will have hash of ids?

Also, I would prefer to get this refactoring done in a separate PR.